### ISSUE TITLE
BalenaOS 2.80.3+rev1 / Supervisor v12.7.0 is not able to provision

### ISSUE THREAD

**Issue by bartversluijs** [balena-io/open-balena#120]
Hi,

As [posted on the forums](https://forums.balena.io/t/device-is-not-able-to-provision/328107), a device with BalenaOS version 2.80.3+rev1 and supervisor v2.17.0 is not able to provision to the openBalena server.

Tried it with a clean install of openBalena, but it didn't work.
I have the supervisor's log of when this happened:
```
[event]   Event: Device bootstrap {}
[info]    New device detected. Provisioning...
[event]   Event: Device bootstrap failed, retrying {"delay":30000,"error":{"message":"Device weathered-sun is too old to satisfy required version range: &gt;=2.11.0","stack":"ApiError: Device weathered-sun is too old to satisfy required version range: &gt;=2.11.0\n    at Object.<anonymous> (/usr/src/app/dist/app.js:10:452890)\n    at Generator.next (<anonymous>)\n    at fulfilled (/usr/src/app/dist/app.js:10:1915792)\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)"}}
```

---

**openBalena details**:
API version: 0.119.5
DB version: 4.1.0
Registry version: 2.16.0
S3 version: 2.9.9
VPN version: 9.17.4

**Device that I’m trying to provision**:
Raspberry Pi 4
OS: Version 2.80.3+rev1 (dev)
Supervisor: v2.17.0

**Comment by dfunckt** [comment]
> Supervisor: v2.17.0

This is fishy -- the Supervisor is way past v2. Are you sure this is the Supervisor version contained in the image? How did you find out?

**Comment by bartversluijs** [comment]
Hmm, I think something went wrong there. But I'll check if something went wrong while copy-pasting or if there's something wrong with the image. Should indeed explain much.

**Comment by bartversluijs** [comment]
Checked it, it was supervisor v12.7.0, OS version 2.80.3+rev1 (development version). Did the same as before, started new cluster and flashed new device, same result.

**Comment by openedhardware** [comment]
Same issue with me.
Any update here? :)

**Comment by bartversluijs** [comment]
Have some news regarding this.
I've created another openBalena instance and changed the API from version `v0.119.5` (which is the default now) to `v0.138.7` from PR #114. This caused the device to provision correctly without any issue. 

Tested with a Raspberry Pi 4, BalenaOS 2.80.5+rev1, supervisor v12.8.7 (development image)

**Comment by openedhardware** [comment]
How did you update the API version on your openBalena server? @bartversluijs 

Thanks!

**Comment by bartversluijs** [comment]
@openedhardware I'm using my own Kubernetes Helm chart, but I've checked what's the best way for the standard Docker-way, and that's by adding the following snippet to `config/docker-compose.yml`

```yaml
services:
  api:
    image: balena/open-balena-api:v0.138.7
```

Because the PR is approved but there are some issues with the commit messages, I think v0.138.7 is safe to use.

**Comment by openedhardware** [comment]
Hmm, it doesn't work.. :/

**Comment by bartversluijs** [comment]
Can you explain what you did?
I'm currently busy with testing the v0.138.7 release and I've just provisioned another Raspberry Pi 4 with BalenaOS 2.80.5+rev1 again with success.

**Comment by bartversluijs** [comment]
As far as I know this is fixed by PR #123.
Tested it with BalenaOS 2.80.3+rev1 & 2.80.5+rev1 on Raspberry Pi 3 and Raspberry Pi 4.
